tfhe/src/core_crypto/entities/glwe_ciphertext.rs

```diff
@@ -321,7 +321,7 @@ where
 {
     data: C,
     polynomial_size: PolynomialSize,
-    ciphertext_modulus: CiphertextModulus<C::Element>,
+    pub(crate) ciphertext_modulus: CiphertextModulus<C::Element>,
```

Comment:
no, there should be an accessor for this field, it stays private, otherwise anybody can mutate it and make inconsistent entities